core/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -52,7 +52,8 @@ object ToolUtils extends Logging {
     "311" -> new ComparableVersion("3.1.1"), // default build version
     "320" -> new ComparableVersion("3.2.0"), // introduced reusedExchange
     "331" -> new ComparableVersion("3.3.1"), // used to check for memoryOverheadFactor
-    "340" -> new ComparableVersion("3.4.0")  // introduces jsonProtocolChanges
+    "340" -> new ComparableVersion("3.4.0"),  // introduces jsonProtocolChanges
+    "350" -> new ComparableVersion("3.5.0")  // introduces windowGroupLimit
```

Comment:
Note: this is not related to this PR; but I thought that we might start thinking of defining the entries of the MAp by pulling the variable-names from the POM file. Just a thought no related to this code change here.

Reply:
Yes. This hard coding is not scalable.